build.gradle

```diff
@@ -49,8 +49,7 @@ configurations {
 
 dependencies {
     implementation 'org.jetbrains:annotations:23.0.0'
-    priority 'org.spigotmc:spigot-api:1.8.8-R0.1-SNAPSHOT'
-    implementation files('lib/craftbukkit-1.8.8-R0.1-SNAPSHOT.jar')
+    priority 'org.spigotmc:spigot-api:1.16.5-R0.1-SNAPSHOT'
     implementation files('lib/NPC-API-0.1.2.jar')
```

Comment:
Remoe that one too `implementation files('lib/NPC-API-0.1.2.jar')` since I'm not gonna port my NPC api but rework it using the new NMS Entity system I'm working on, we'll use regular villager while I redo NPCs